### PR TITLE
[MIG] stock_picking_group_by_partner_by_carrier: Migration to 14.0

### PR DESCRIPTION
**Pull request by phuctranfxvn**
Depends on:
- stock_move_assign_picking_hook https://github.com/OCA/stock-logistics-workflow/pull/796

### CODE REVIEW

File: stock_picking_group_by_partner_by_carrier/__manifest__.py [hunk 1]
```diff
@@ -4,7 +4,7 @@
 {
     "name": "Stock Picking: group by partner and carrier",
     "Summary": "Group sales deliveries moves in 1 picking per partner and carrier",
-    "version": "13.0.1.6.1",
+    "version": "14.0.1.6.1",
```
Comment (simahawk) [inline stock_picking_group_by_partner_by_carrier/__manifest__.py:7]: 14.0.1.0.0

File: test-requirements.txt [hunk 2]
```diff
@@ -0,0 +1 @@
+odoo14-addon-stock_move_assign_picking_hook @ git+https://github.com/phuctranerp/stock-logistics-workflow.git@14.0-mig-stock_move_assign_picking_hook#subdirectory=setup/stock_move_assign_picking_hook
```
Comment (simahawk) [inline test-requirements.txt:1]: to clean up

**Review by sebalix — APPROVED** [review]
LG, need to address @simahawk 's comment however

**Comment by simahawk** [comment]
https://github.com/OCA/stock-logistics-workflow/pull/874

**Pull request closed without merging**